src/api/version.js

```diff
@@ -29,6 +29,23 @@ const versionApi = {
     }, (res) => {
       return Promise.reject(res);
     });
+  },
+
+  /**
+   * Get version of full node running in connected server
+   *
+   * @return {Promise}
+   * @memberof ApiVersion
+   * @inner
+   */
+  async asyncGetVersion() {
+    return new Promise((resolve, reject) => {
+      createRequestInstance(resolve).get(`version`).then((res) => {
+        resolve(res.data);
+      }, (res) => {
```

Comment:
:star2:  This wrapper is a welcome addition!

I would only suggest calling this object `err` , to reinforce it is not a regular result.
```suggestion
      }, (err) => {
```

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-lib/pull/579/commits/9951ac7161a6ea17df875f0a8e2d9eb258d4c99a